extensions/workspace/watch/output-formatter.ts

```diff
@@ -0,0 +1,40 @@
+import chalk from 'chalk';
+
+export const formatWatchPathsSortByComponent = (watchPathsSortByComponent) => {
+ return watchPathsSortByComponent.reduce((outputString, watchPath) => (
+    outputString + 
```

Comment:
do not mix the + and `` syntax
you also probably don't need the \n
probably better to be something like this (not tested):
```js
`${outputString}
 ${chalk.green('√')} SUCCESS\t${watchPath.componentId}  
      - ${watchPath.absPaths?.join('\n')}
  ), ` ${chalk.underline('STATUS       COMPONENT ID')}
`) 
```
You can also use padleft / padright for the alignments

Reply:
Done.